x/concentrated-liquidity/swaps_test.go

```diff
@@ -1361,3 +1243,88 @@ func (s *KeeperTestSuite) TestSwapExactAmountOut() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) TestInverseRelationshipSwapOutAmtGivenIn() {
+	tests := swapOutGivenInCases
+
+	for name, test := range tests {
+		s.Run(name, func() {
+			s.Setup()
+			s.FundAcc(s.TestAccs[0], sdk.NewCoins(sdk.NewCoin("eth", sdk.NewInt(10000000000000)), sdk.NewCoin("usdc", sdk.NewInt(1000000000000))))
+			s.FundAcc(s.TestAccs[1], sdk.NewCoins(sdk.NewCoin("eth", sdk.NewInt(10000000000000)), sdk.NewCoin("usdc", sdk.NewInt(1000000000000))))
+
+			// Create default CL pool
+			pool := s.PrepareConcentratedPool()
+
+			// add default position
+			_, _, _, err := s.App.ConcentratedLiquidityKeeper.CreatePosition(s.Ctx, pool.GetId(), s.TestAccs[0], DefaultAmt0, DefaultAmt1, sdk.ZeroInt(), sdk.ZeroInt(), DefaultLowerTick, DefaultUpperTick)
```

Comment:
nit: we might have a helper to create a default position

Reply:
nice catch, changed